ansible/roles/postgresql-data-update-managed/tasks/main.yml

```diff
@@ -39,3 +41,21 @@
     name: "{{ enc_postgres_database }}"
     state: restore
     target: "/tmp/enc_postgres.sql"
+
+- name: Create the schema for uci transformer-service
```

Comment:
Please add the same task in `postgresql-data-update` also.

Reply:
@keshavprasadms - updated the tasks in `postgresql-data-update` role.